dockerfiles/postgres/init-debug.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I think it also should be reverted since it is supposed to be executed in docker container.